core/server/web/oauth/app.js

```diff
@@ -106,12 +103,11 @@ module.exports = function setupOAuthApp() {
 
                 await auth.session.sessionService.createSessionForUser(req, res, req.user);
 
-                return res.redirect('/ghost/');
+                return res.redirect(urlUtils.urlFor('admin'));
             }), {
                 scope: ['profile', 'email'],
                 session: false,
-                prompt: 'consent',
-                accessType: 'offline'
```

Comment:
Why was the `accessType: 'offline'` parameter removed? IIRC without this parameter we won't get a refresh token in the response :thinking:

Reply:
I'd removed it to stop my editor complaining and then forgot to add it back - fixed!